src/main/resources/com/google/api/codegen/php/client_impl.snip

```diff
@@ -318,65 +319,36 @@
      @else
          *     Optional. Options for configuring the service API wrapper.
      @end
-     *
      @if xapiClass.hasDefaultServiceAddress
-         *     @@type string $serviceAddress The domain name of the API remote host.
-         *                                  Default '{@xapiClass.serviceAddress}'.
+         *     @@type string $serviceAddress The address of the API remote host. Must be formatted as "<uri>:<port>".
+         *                                   Default '{@xapiClass.serviceAddress}'.
      @else
-         *     @@type string $serviceAddress Required. The domain name of the API remote host.
-     @end
-     *     @@type mixed $port The port on which to connect to the remote host. Default {@xapiClass.servicePort}.
-     *     @@type Channel $channel
-     *           A `Channel` object. If not specified, a channel will be constructed.
-     *           NOTE: This option is only valid when utilizing the gRPC transport.
-     *     @@type ChannelCredentials $sslCreds
-     *           A `ChannelCredentials` object for use with an SSL-enabled channel.
-     *           Default: a credentials object returned from
-     *           \Grpc\ChannelCredentials::createSsl().
-     *           NOTE: This option is only valid when utilizing the gRPC transport. Also, if the $channel
-     *           optional argument is specified, then this argument is unused.
-     *     @@type bool $forceNewChannel
-     *           If true, this forces gRPC to create a new channel instead of using a persistent channel.
-     *           Defaults to false.
-     *           NOTE: This option is only valid when utilizing the gRPC transport. Also, if the $channel
-     *           optional argument is specified, then this option is unused.
-     *     @@type CredentialsLoader $credentialsLoader
-     *           A CredentialsLoader object created using the Google\Auth library.
-     @if xapiClass.hasDefaultServiceScopes
-         *     @@type string[] $scopes A string array of scopes to use when acquiring credentials.
-         *                          Defaults to the scopes for the {@xapiClass.serviceTitle}.
-     @else
-         *     @@type string[] $scopes Required. A string array of scopes to use when acquiring credentials.
+         *     @@type string $serviceAddress Required. The address of the API remote host. Must be formatted as "<uri>:<port>".
      @end
+     *     @@type array $keyFile The contents of the service account credentials
+     *            .json file retrieved from the Google Developer's Console.
+     *            Ex: `json_decode(file_get_contents($path), true)`.
+     *     @@type string $keyFilePath The full path to your service account
+     *            credentials .json file retrieved from the Google Developers
+     *            Console.
+     *     @@type string $clientConfig
+     *            Array containing client method configuration, including retry settings. If this argument
```

Comment:
Does this documentation come from somewhere (like .proto file)? There are few inconsistencies in this documentation (like different maximum right margin for top and bottom fields (bottom is much wider), .json vs JSON when talking about json files etc. If it is external, then ok, if it is not, please make it moreconsistent.